2.0.1/buster/Dockerfile

```
@@ -0,0 +1,188 @@
+
+ENV ASTRONOMER_USER=${ASTRONOMER_USER}
+ENV ASTRONOMER_UID=${ASTRONOMER_UID}
+
+# Need to repeat the empty argument here otherwise it will not be set for this stage
+# But the default value carries from the one set before FROM
+ARG PYTHON_BASE_IMAGE
+ENV PYTHON_BASE_IMAGE=${PYTHON_BASE_IMAGE}
+
+# Make sure noninteractie debian install is used and language variables set
+ENV DEBIAN_FRONTEND=noninteractive LANGUAGE=C.UTF-8 LANG=C.UTF-8 LC_ALL=C.UTF-8 \
+    LC_CTYPE=C.UTF-8 LC_MESSAGES=C.UTF-8
+
+# By increasing this number we can do force build of all dependencies
+ARG DEPENDENCIES_EPOCH_NUMBER="1"
+# Increase the value below to force renstalling of all dependencies
+ENV DEPENDENCIES_EPOCH_NUMBER=${DEPENDENCIES_EPOCH_NUMBER}
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+           apt-utils \
+           curl \
+           libmariadb3 \
+           freetds-bin \
+           gosu \
+           libffi6 \
+           libkrb5-3 \
+           libpq5 \
+           libsasl2-2 \
+           libsasl2-modules \
+           libssl1.1 \
+           locales  \
+           netcat \
+           rsync \
+           sasl2-bin \
+           sudo \
+           tini \
+    && apt-get autoremove -yqq --purge \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN pip install --upgrade pip==19.3.1
+
+RUN useradd --uid $ASTRONOMER_UID --create-home ${ASTRONOMER_USER} \
+    && groupadd astrogroup --gid 101 \
+    && usermod --append --groups astrogroup ${ASTRONOMER_USER}
+
+# From the airflow image on master
+
+#################################################################
+######## Installed dependencies - now installing Airflow ########
+#################################################################
+
+FROM ${APT_DEPS_IMAGE} as devel
+SHELL ["/bin/bash", "-o", "pipefail", "-e", "-u", "-x", "-c"]
+
+ENV PIP_NO_CACHE_DIR="true"
+
+RUN apt-get update \
+    && curl -sL https://deb.nodesource.com/setup_13.x | sudo bash - \
+    && apt-get install -y --no-install-recommends \
+        build-essential \
+        default-libmysqlclient-dev \
+        libffi-dev \
+        libkrb5-dev \
+        libpq-dev \
+        libsasl2-dev \
+        libssl-dev \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+ARG VERSION="2.0.1-1.*"
+ARG SUBMODULES="async,azure,amazon,celery,elasticsearch,google,password,cncf.kubernetes,mysql,postgres,redis,slack,ssh,statsd,virtualenv"
+ARG AIRFLOW_MODULE="astronomer_certified[${SUBMODULES}]==$VERSION"
+ARG AIRFLOW_VERSION="2.0.0"
+
+# Make pip look at our pip repo too, and force it to install these specific
+# versions when ever it installs a module.
+COPY include/pip.conf /etc/pip.conf
+COPY include/pip-constraints.txt /usr/local/share/astronomer-pip-constraints.txt
+
+# Pip install airflow and astro security manager
+RUN pip install "${AIRFLOW_MODULE}" --constraint "https://raw.githubusercontent.com/apache/airflow/constraints-${AIRFLOW_VERSION}/constraints-3.7.txt" \
```

For now you can use this constraints file

```suggestion
RUN pip install "${AIRFLOW_MODULE}" --constraint "https://raw.githubusercontent.com/apache/airflow/constraints-2.0/constraints-3.7.txt" \
```